library/Zend/Mail/Message.php

```diff
@@ -36,6 +36,44 @@ class Message
     protected $encoding = 'ASCII';
 
     /**
+     * Constructor.
+     *
+     * @param null|array|Traversable $options (Optional) Options to set
+     * @param array|Traversable $options
+     */
+    public function __construct($options = null)
```

Comment:
I don't see the feature you reference. Can you explain me ?
In original constructor, if [$mail is an array](https://github.com/zendframework/zf2/blob/master/library/Zend/Log/Writer/Mail.php#L82), we call the parent constructor (seek for filters and formatter) and then [get $mail['mail']](https://github.com/zendframework/zf2/blob/master/library/Zend/Log/Writer/Mail.php#L88). After, the constructor check that [$mail is an Zend\Mail\Message instance](https://github.com/zendframework/zf2/blob/master/library/Zend/Log/Writer/Mail.php#L92).

When does the Zend\Mail\Message instance is created ?
I don't see any plugin manager or factory involved here.

How can i configure a log writer which sends an email to administrator ?

(Sorry for the late reply. I saw your reply few days ago only)